src/main/java/model/lotto/RandomLottoNumbersGenerator.java

```diff
@@ -0,0 +1,27 @@
+package model.lotto;
+
+import model.lotto.LottoNumbers;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RandomLottoNumbersGenerator {
+    private static final int LOTTO_SIZE = 6;
+
+    public static List<Integer> pickSixNumbers() {
+        List<Integer> shuffledNumbers = shuffleLottoNumbers();
+        return shuffledNumbers.stream()
+                .limit(LOTTO_SIZE)
+                .collect(Collectors.toList());
+    }
+
+    private static List<Integer> shuffleLottoNumbers() {
+        ArrayList<Integer> randomNumbers = new ArrayList<>();
+        randomNumbers.addAll(LottoNumbers.getLottoNumbers());
```

Comment:
아래처럼 한 줄로 줄일 수도 있겠네요.

```suggestion
        ArrayList<Integer> randomNumbers = new ArrayList<>(LottoNumbers.getLottoNumbers());
```